rtc/RobotHardware/RobotHardware.cpp

```diff
@@ -61,7 +62,8 @@ RobotHardware::RobotHardware(RTC::Manager* manager)
     m_rstate2Out("rstate2", m_rstate2),
     m_RobotHardwareServicePort("RobotHardwareService"),
     // </rtc-template>
-	dummy(0)
+    allowTorqueControlMode(true),
```

Comment:
Is this necessary? There is no way to change its value.

Reply:
Sorry for this, it remained after being used by code which was not ultimately used and that I erased
Solved by https://github.com/fkanehiro/hrpsys-base/pull/1326/commits/d06803392e4ea20a2be220bba0512397ba96069c